docs/application/native/guides/app-management/ui-app/efl-app.md

```diff
@@ -10,19 +10,24 @@ EFL is a collection of libraries that are independent or can build on top of eac
 Before you start UI programming with EFL, you must familiarize yourself with the basic EFL fundamentals. EFL consists of the following main libraries:
 
 - **Elementary**
-Elementary provides all the functions you need to create a window, create simple and complex layouts, manage the life-cycle of a view, and add UI components.
+
+  Elementary provides all the functions you need to create a window, create simple and complex layouts, manage the life-cycle of a view, and add UI components.
 
 - **Edje**
-Edje provides a powerful theme library for your application. You can also use Edje to create your own objects and use them in your application, or even extend the default theme.
+
+  Edje provides a powerful theme library for your application. You can also use Edje to create your own objects and use them in your application, or even extend the default theme.
 
 - **Ecore**
-Ecore manages the main loop of your application. The main loop is where events are handled, and where you interact with the user through the callback mechanism.
+
+  Ecore manages the main loop of your application. The main loop is where events are handled, and where you interact with the user through the callback mechanism.
 
 - **Evas**
-- Evas is the canvas engine responsible for managing the drawing of your content. All graphical objects that you create are Evas objects.
+
+  Evas is the canvas engine responsible for managing the drawing of your content. All graphical objects that you create are Evas objects.
 
 - **Eina**
-- Eina is a toolbox that implements an API for data types in an efficient way. It contains all the functions needed to create lists and hashes, manage shared strings, open shared libraries, and manage errors and memory pools.
+
+  Eina is a toolbox that implements an API for data types in an efficient way. It contains all the functions needed to create lists and hashes, manage shared strings, open shared libraries, and manage errors and memory pools.
 
 For more information on how to get started with EFL programming, see [Getting Started with EFL UI Programming](../ui/efl/getting-started.md). It helps you to understand how EFL works, and how you can create a simple EFL application with basic features.
```

Comment:
The link `[Getting Started with EFL UI Programming](../ui/efl/getting-started.md)` is broken. `(../../ui/efl/getting-started.md)` looks correct

Reply:
done